### ISSUE TITLE
NTR: Muscle biopsy: debranching enzyme activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9073]
New term request Muscle biopsy: debranching enzyme activity low (segal_050203152120):

**Comment by pnrobinson** [comment]
@MickeySegal
does this refer to https://omim.org/entry/610860 ?

**Comment by MickeySegal** [comment]
That is the gene for the enzyme, but the finding refers to testing for the protein product.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000616
- label: Reduced muscle glycogen debrancher enzyme activity
- parents: [Abnormal enzyme activity in muscle tissue (HP:0034526)](https://hpo.jax.org/app/browse/term/HP:0034526)
- pmid: PMID:10472540
- def: Activity of the gycogen debrancher enzyme (GDE) in muscle tissue is below the lower limit of normal. GDE is a large monomeric protein with two catalytic activities: amylo-1,6-glucosidase (EC 3.2.1.33) and 4-alpha-glucanotransferase (EC 2.4.1.25).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9073
- synonyms:
